src/modules/FunctionActivator.js

```diff
@@ -0,0 +1,23 @@
+import createGame from './PostFunction.js';
+import scoreBox from './scoreBox.js';
+
+const form = document.querySelector('form');
+const userName = document.getElementById('name-input');
+const userScore = document.getElementById('score-input');
+const refresh = document.querySelector('.refresh');
+
+function postActivator() {
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    createGame(userName.value, userScore.value);
+    form.reset();
+  });
+}
+
+function getActivator() {
+  refresh.addEventListener('click', async () => {
+    await scoreBox();
+  });
+}
```

Comment:
- Based on the project requirement you are expected to use arrow functions across your code base. Kindly refactor this function to use the arrow `() => ` function.